app/src/main/res/values/strings.xml

```diff
@@ -446,6 +445,22 @@
   <string name="ratio_default_hint_text" description="The default hint text for the ratio interaction if the placeholder text is not available.">
     Enter a ratio in the form x:y.
   </string>
+  <!-- DeveloperOptionsFragment -->
+  <string name="developer_options_activity_title">Developer Options</string>
```

Comment:
How does this PR help the codebase? Asking this because I don't see any usage of these strings in project which means that they are unused and not required.